content/3_Silver/Prefix_Sums.mdx

```diff
@@ -293,6 +293,19 @@ for i in range(Q):
 
 </LanguageSection>
 
+## Difference Array
+
+<FocusProblem problem="haybale-stacking" />
+
+<Spoiler title = "Hint">
+What if you did prefix sums, but in reverse?
+</Spoiler>
+
+<Spoiler title = "Solution">
+Consider the array $[0, 1, 0, 0, 0]$. Running prefix sums on this gives us $[0, 1, 1, 1, 1]$. Using prefix sums, we can implicitly add to a suffix by changing one number.
```

Comment:
include a better example with 1s as well as -1s.
state that this is equivalent to a range addition